src/main/java/io/horrorshow/codey/time/DiscordTimezone.java

```diff
@@ -0,0 +1,135 @@
+package io.horrorshow.codey.time;
+
+import io.horrorshow.codey.discordutil.DiscordUtils;
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.JDA;
+import net.dv8tion.jda.api.entities.Message;
+import net.dv8tion.jda.api.events.message.guild.GuildMessageReceivedEvent;
+import net.dv8tion.jda.api.hooks.ListenerAdapter;
+import org.jetbrains.annotations.NotNull;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.time.OffsetDateTime;
+import java.time.ZoneId;
+import java.util.Arrays;
+
+@Service
+public class DiscordTimezone extends ListenerAdapter {
+    private final DiscordUtils utils;
+
+    public DiscordTimezone(@Autowired JDA jda, @Autowired DiscordUtils utils) {
+        this.utils = utils;
+
+        jda.addEventListener(this);
+    }
+
+    private final String emoji = "⏰";
+    private final String deleteEmoji = "\uD83D\uDDD1";
```

Comment:
this shouldn't be needed anymore once the message sending / replying is in DiscordUtils (see the comment below)